src/site/apt/examples/multi-module-config.apt.vm

```diff
@@ -27,6 +27,10 @@
 
 Multimodule Configuration
 
+  <<Since version 2.12 of this Plugin you can also
```

Comment:
I am not sure this section is really useful here

Any way using **bold** seems to "strong" to me

Reply:
I included this section because there are now 2 ways to inherit the config.
Either put it into a .jar (the way described here) or use the inline config.

A user looking for a way to inherit the config will probably click on the multi-module-config page but not on the inline page.

Thats why I cross referenced it here.

Reply:
good point.
loading into CI